packages/charts/src/components/ComposedChart/index.tsx

```diff
@@ -267,11 +275,18 @@ const ComposedChart: FC<ComposedChartProps> = forwardRef((props: ComposedChartPr
             {...measureAxisProps}
             yAxisId="primary"
             width={yAxisWidth}
+            orientation={useIsRTL() ? 'right' : 'left'}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.